integration/test/native_installer_test.go

```diff
@@ -1,4 +1,5 @@
-//+build install_native
+//go:build install_native
```

Comment:
adding them back? they were just removed?

Reply:
yep, I planned on merging into this one after CI runs